src/esp/assets/ResourceManager.cpp

```diff
@@ -1192,13 +1193,16 @@ void ResourceManager::loadMaterials(Importer& importer,
     if (loadedAssetData.assetInfo.requiresLighting &&
         materialData->types() &
             Magnum::Trade::MaterialType::PbrMetallicRoughness) {
-      const Mn::Trade::PbrMetallicRoughnessMaterialData&
-          pbrMetallicRoughnessMaterialData =
-              static_cast<Mn::Trade::PbrMetallicRoughnessMaterialData&>(
-                  *materialData);
+      const auto& pbrMaterialData =
+          materialData->as<Mn::Trade::PbrMetallicRoughnessMaterialData>();
 
-      finalMaterial = gfx::buildPhongFromPbrMetallicRoughness(
-          pbrMetallicRoughnessMaterialData, textureBaseIndex, textures_);
```

Comment:
@eundersander :

Is it a bug here? Is it missing a `continue`?
Here if it requires lighting and the material is PBR, then it will initialize the `finalMaterial` as Phong material using PBR material. It should skip the rest of the code and continue the next loop. Correct?
Otherwise the following code will build the `finalMaterial` as `PhongShaderMaterialData` anyway.